src/latexgen.cpp

```diff
@@ -373,7 +377,7 @@ static void writeLatexMakefile()
   t << endl
     << "clean:" << endl
     << "\trm -f " 
-    << "*.ps *.dvi *.aux *.toc *.idx *.ind *.ilg *.log *.out *.brf *.blg *.bbl refman.pdf" << endl;
+    << "rm -f $(CLEAN_FILES)" << endl;
```

Comment:
Look a bit double here, shouldn't this just be `\t"rm -f $(CLEAN_FILES)" << endl;`

Reply:
yes... :-/